code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -323,7 +326,12 @@
 
 /mob/living/silicon/robot/drone/start_pulling(var/atom/movable/AM)
 
-	if(istype(AM,/obj/item/pipe) || istype(AM,/obj/structure/disposalconstruct))
+	if(istype(AM,/obj/item/pipe) \
+	|| istype(AM,/obj/structure/disposalconstruct) \
+	|| istype(AM,/obj/item/stack/cable_coil) \
+	|| istype(AM,/obj/item/stack/rods) \
+	|| istype(AM,/obj/item/stack/sheet) \
+	|| istype(AM,/obj/item/stack/tile))
```

Comment:
I suggest making a list of all these paths, something like `pullable_drone_items`, so you can do
 `if(is_type_in_list(AM, pullable_drone_items))`.

Reply:
Where should I put the list?

Reply:
I was thinking as a var on drones.